datafusion/physical-plan/src/windows/window_agg_exec.rs

```diff
@@ -386,12 +382,19 @@ impl WindowAggStream {
                     continue;
                 }
                 Some(Err(e)) => Err(e),
-                None => self.compute_aggregates(),
-            };
+                None => {
+                    let batch = concat_batches(&self.input.schema(), &self.batches)?;
```

Comment:
This change moves `concat_batches` out of `elapsed_compute`

Reply:
I think we should fix this prior to merge